DesignSafe-CI/portal
# Change HPC Scratch to HPC Work

## Overview: ##

Instead of a user-scratch being mounted in interactive sessions, a user-work will be used instead.

⚠️ Important: `AGAVE_WORKING_SYSTEM` must be updated to `"designsafe.storage.frontera.work"` in `designsafe.env` and in ansible group vars on deployment

## PR Status: ##


## Testing Steps: ##
1. Update `AGAVE_WORKING_SYSTEM` as stated above
2. Login to designsafe.dev
3. Confirm you can list and create files at https://designsafe.dev/data/browser/agave/designsafe.storage.frontera.work/$username/
